extensions/canvasMedical/v1/actions/index.ts

```diff
@@ -0,0 +1,14 @@
+export { createAppointment } from './createAppointment'
+export { updateAppointment } from './updateAppointment'
+export { createTask } from './createTask'
+export { updateTask } from './updateTask'
+export { createPatient } from './createPatient'
+export { updatePatient } from './updatePatient'
+export { createQuestionnaireResponses } from './createQuestionnaireResponses'
+
+// Note: should also NOT be exported, since that is only used with the JSON type data points, which we don't support yet
+// export { getAppointment } from './getAppointment'
+// export { getTask } from './getTask'
+// export { getPatient } from './getPatient'
+// export { extractPatientInfo } from './extractPatientInfo'
+// export { getQuestionnaireResponse } from './getQuestionnaireResponse'
```

Comment:
okay I think I get the misunderstanding now @kubiak85 -- JSON data points are supported in the Awell platform but what this exactly means was not communicated well. Sorry about that.
See this for explanation: https://awellhealth.slack.com/archives/C04RZELCC1G/p1693564019387499?thread_ts=1693555494.973049&cid=C04RZELCC1G

The FieldType determines two things:
- the default manual input
- the data point types supported in the optional data point selector

When choosing STRING for example, this means the default input is a simple text input, but you can switch to a selector and choose a data point saved in the system (e.g. patient first name) that represents a string (so we filter out dates, numbers, etc).

When choosing a JSON, the same should happen ideally: default is a JSON editor and you can construct an object (which we have today), and the alternative is a Select component and you can choose data points that are storing JSON-parseable content (e.g. objects or arrays). However this second one we don't support today, so when you choose FieldType.JSON we just show the JSON object editor and there is NO option to switch to a selector. This is good, and what we want so you can safely use FieldType.JSON as the type for all the fields where we want the user to construct the JSON object.

The `jsonType` field should still be removed (/not added back in) because that is related to the future support for JSON data points, and is not implemented yet.

Reply:
@rahulkeerthi Just FYI, we don't have access to the link you provided (https://awellhealth.slack.com/archives/C04RZELCC1G/p1693564019387499?thread_ts=1693555494.973049&cid=C04RZELCC1G)

But I believe all is clear

Reply:
@michal-grzelak oh that is weird, it was the message from our conversation 
![image](https://github.com/awell-health/awell-extensions/assets/2485818/c6585355-a582-4b88-b43f-49f0217ae8b7)